OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -47,6 +47,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			register("all", "toggles all cheats and gives you some cash for your trouble.");
 			register("crash", "crashes the game.");
 			register("levelup", "adds a specified number of levels to the selected actors.");
+			register("toggleui", "toggles the ui (disables pausing) for a better view.");
```

Comment:
`(disables pausing)`? You can still pause the game. I think you meant `(disables the menu)`?